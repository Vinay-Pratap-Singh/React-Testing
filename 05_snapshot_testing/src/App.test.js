const { render } = require("@testing-library/react");
const { default: App } = require("./App");

test("snapshot testing", () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
