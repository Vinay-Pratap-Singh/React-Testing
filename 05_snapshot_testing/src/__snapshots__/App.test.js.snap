// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing 1`] = `
<div>
  <div>
    <h1>
      Snapshot testing
    </h1>
    <p>
      Performing the snapshot testing to check the component ui change
    </p>
  </div>
</div>
`;
